package/gluon-status-page/Makefile

```diff
@@ -28,6 +28,12 @@ define Download/almond
 	MD5SUM:=aa66c0c0cb55a4627bb706df73f3aff5
 endef
 
+define Download/EventSource
+	FILE:=eventsource.js
+	URL:=https://raw.githubusercontent.com/altima/EventSource/master/eventsource.js
```

Comment:
The URL should be (pin it to a tag, use upstream repo URL, don't include filename):

    https://raw.githubusercontent.com/Yaffle/EventSource/0.0.8

Also, you need to add `$(eval $(call Download,EventSource))` below.